magento/magento2
# Missed form validation in Admin Order Address Edit route sales/order/address

Original Issue https://github.com/magento/magento2/issues/19360

### Description (*)
Magento 2.2.* and 2.3.*
EAV validation rules are not checked during editing an address in order.
route: sales/order/address
See column validate_rules in table customer_eav_attribute
e.g. name may have '{"max_text_length":255}'
Controller: \Magento\Sales\Controller\Adminhtml\Order\AddressSave::execute

This rules work in other forms on fronted and in address form and during creating new order from Admin
route: sales/order_create/index/

Note: added classes for inputs according to validation rules for attribute

### Manual testing scenarios (*)
Create an order
Find the order in admin
Click edit on address section
Fill the field firstname or lastname with any characters till length be more than 255
Save address and see saved value

### Contribution checklist (*)
